desktop/app.js

```diff
@@ -52,6 +53,19 @@ module.exports = function main() {
       show: false,
     });
 
+    // Tray
+    const tray = new Tray(iconPath)
```

Comment:
@dmsnell I created a deb package and found out that this icon path doesn't exist in the final installation dir (`/usr/share/simplenote`). The icon is actually placed at `/usr/share/pixmaps/simplenote.png`. Should I reference this path here instead?